src/components/sidebar/Sidebar.js

```diff
@@ -9,44 +9,66 @@ const filters = [
     type: "Category",
     list: ["All items", "Handmade", "Vintage"]
```

Comment:
Suggestion- if you are using mockbee and the categories data from mockbee, you could fetch that and use here. This way any change in categories, you need not make the change seprately in home page (where you render the categories) and sidebar/ product listing page.

Reply:
yes, I do plan to update it in the later iteration along with other backend integration